### ISSUE TITLE
Unable to compile 1.2.16 (or newer) on Alpine Linux due to missing backtrace

### ISSUE THREAD

**Issue by ctrowat** [Cacti/spine#201]
During the configure process it does a test for whether backtrace is available

```
...
checking whether to verify net-snmp library vs header versions... no
checking if we can support backtracing... no
checking that generated files are newer than configure... done
...
```

However later it compiles as if backtrace was available:
```
config.status: executing libtool commands
/bin/sh ./libtool  --tag=CC   --mode=link gcc  -I/usr/include/net-snmp -I/usr/include/net-snmp/.. -I/usr/include/mysql -g -O2  -L/usr/lib -L/usr/lib  -o spine sql.o spine.o util.o snmp.o locks.o poller.o nft_popen.o php.o ping.o keywords.o error.o  -lnetsnmp -lmysqlclient_r -lm -ldl -lmysqlclient -lm -ldl -lcrypto -lz -lpthread -ldl -lm -lpthread -lssl
libtool: link: gcc -I/usr/include/net-snmp -I/usr/include/net-snmp/.. -I/usr/include/mysql -g -O2 -o spine sql.o spine.o util.o snmp.o locks.o poller.o nft_popen.o php.o ping.o keywords.o error.o  -L/usr/lib -lnetsnmp -lmysqlclient_r -lmysqlclient -lcrypto -lz -ldl -lm -lpthread -lssl
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: error.o: in function `spine_print_backtrace':
/tmp/cacti-spine-1.2.16/error.c:99: undefined reference to `backtrace'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/cacti-spine-1.2.16/error.c:105: undefined reference to `backtrace_symbols'
collect2: error: ld returned 1 exit status
make: *** [Makefile:475: spine] Error 1
```

The addition of backtrace to the error output relies on glibc, which is not available on Alpine Linux.  Could the build process honour the decision made when configuring the build?  Or could the reliance on backtrace perhaps be replaced with something else?

**Comment by TheWitness** [comment]
This issue should be resolved in the 1.2.x branch.  So, please test using that:

```bash
git clone -b 1.2.x https://github.com/cacti/spine.git
```

**Comment by TheWitness** [comment]
No feedback.
